File: launchdarkly/typescript-node/src/GetInsightGroupExample.ts
import * as fs from 'fs';
import api from "launchdarkly_client"
import models from "launchdarkly_client"

const apiCaller = new api.InsightsScoresBetaApi();
apiCaller.setApiKey(api.InsightsScoresBetaApiApiKeys.ApiKey, "YOUR_API_KEY");

apiCaller.getInsightGroup(
  undefined, // insightGroupKey
  undefined, // expand
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling InsightsScoresBetaApi#getInsightGroup:");
  console.log(error.body);
});

File: launchdarkly/kotlin/src/main/kotlin/oseg/launchdarkly_examples/GetInsightGroupExample.kt
package oseg.launchdarkly_examples

import com.launchdarkly.client.infrastructure.*
import com.launchdarkly.client.apis.*
import com.launchdarkly.client.models.*

import java.io.File
import java.time.LocalDate
import java.time.OffsetDateTime
import kotlin.collections.ArrayList
import kotlin.collections.List
import kotlin.collections.Map
import com.squareup.moshi.adapter

@ExperimentalStdlibApi
class GetInsightGroupExample
{
    fun getInsightGroup()
    {
        ApiClient.apiKey["Authorization"] = "YOUR_API_KEY"

        try
        {
            val response = InsightsScoresBetaApi().getInsightGroup(
                insightGroupKey = "insightGroupKey_string",
                expand = null,
            )

            println(response)
        } catch (e: ClientException) {
            println("4xx response calling InsightsScoresBetaApi#getInsightGroup")
            e.printStackTrace()
        } catch (e: ServerException) {
            println("5xx response calling InsightsScoresBetaApi#getInsightGroup")
            e.printStackTrace()
        }
    }
}
